lib/system_win32.h

```diff
@@ -7,7 +7,7 @@
  *                            | (__| |_| |  _ <| |___
  *                             \___|\___/|_| \_\_____|
  *
- * Copyright (C) 2016, Steve Holme, <steve_holme@hotmail.com>.
+ * Copyright (C) 2016 = 2019, Steve Holme, <steve_holme@hotmail.com>.
```

Comment:
s/=/-/

Reply:
Obviously can't type! I blame the new (Microsoft) keyboard !!